scripts/linux/3_pl_server.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-rm -rf /home/xilinx/pynq/bitstream/.log
```

Comment:
@PeterOgden .log file is required in the PL class. If we run into any problem with the PL class socket, this file may not be deleted automatically. So I suggest we still keep `rm -rf .log` statement.